src/python/pants/engine/target_test.py

```diff
@@ -282,33 +279,73 @@ def hydrate_field(
     assert "//:lib" in str(exc)
 
 
+class RustTarget(Target):
```

Comment:
Because we're trying to add Rust as an official language, it would be good to use a different language. I've been using Fortran and Smalltalk in examples. Cobol could work too, hehe.

Reply:
Why play that game? These are locally namespaced. There's no ambiguity or worry of it.

Reply:
I'm fine if you want to keep it, but parroting review feedback I had gotten. https://github.com/pantsbuild/pants/pull/9376#discussion_r396848143